src/components/ViewDeck/ViewDeck.js

```diff
@@ -1,94 +1,77 @@
-import React, { Component } from "react";
+import React, { Component, useState, useEffect } from "react";
 import { connect } from "react-redux";
 import mapStoreToProps from "../../redux/mapStoreToProps";
 import { PieChart } from "react-minimal-pie-chart";
 import "./ViewDeck.css";
 
-class ViewDeck extends Component {
-  state = {
-    hoverCard: "https://i.stack.imgur.com/Vkq2a.png",
-    updatedQty: "",
-    deckQty: "",
-    parsedCards: [],
-  };
+function ViewDeck(props) {
+  const [state, setState] = React.useState({
+    //STILL NEED STATE
+  });
+
+  const [hoverCard, setHoverCard] = useState(
+    "https://i.stack.imgur.com/Vkq2a.png"
+  );
 
-  componentDidMount() {
-    this.props.dispatch({
+  useEffect(() => {
+    props.dispatch({
       type: "GET_LIST",
-      payload: this.props.reduxStore.selectedDeck.id,
+      payload: props.reduxStore.selectedDeck.id,
     });
 
-    this.props.dispatch({
+    props.dispatch({
       type: "GET_SELECTED_DECK",
-      payload: this.props.history.location.pathname.split("/")[2],
+      payload: props.history.location.pathname.split("/")[2],
     });
-  }
+  }, [state]);
 
-  editDeck = (deck) => {
-    console.log("in edit deck with:", deck);
-    const id = this.props.history.location.pathname.split("/")[2];
-    this.props.history.push(`/editdeck/${id}`);
+  const editDeck = (deck) => {
+    const id = props.history.location.pathname.split("/")[2];
+    props.history.push(`/editdeck/${id}`);
```

Comment:
`deck` is an argument that is never used. What is the point pf parsing history? Isn't the `deck.id` what we want to use to navigate?